### PR TITLE
[WIP]adding modified demo-fdmi code to make it compatible with the opensource version

### PR DESCRIPTION
**Pull request by rupasree-roy**
Signed-off-by: Rupasree Roy <rupasree.roy@seagate.com>

### CODE REVIEW

File: demo-fdmi/clovis-app/clovis.map [hunk 1]
```diff
@@ -0,0 +1,492 @@
+Archive member included to satisfy reference by file (symbol)
```
Comment (huanghua78) [inline demo-fdmi/clovis-app/clovis.map:1]: This file is not needed.

**Comment by huanghua78** [comment]
demo-fdmi/clovis-app/src/main
demo-fdmi/clovis-app/src/tags

These files are not needed.

**Comment by huanghua78** [comment]
What is the status of this PR?
Has it passed compilation?
Has it been tested?

File: demo-fdmi/clovis-app/Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+CC=gcc
+
+MERO_PATH=/home/rroy/cortx/cortx-motr
```
Comment (huanghua78) [inline demo-fdmi/clovis-app/Makefile:3]: This should be changed to use the TOPLEVEL source tree.

**Comment by gowthamchinna** [comment]
test this please

**Comment by huanghua78** [comment]
This is closed since it is inactive for a long time.

**Pull request closed without merging**